### PR TITLE
Added api opt-out setting

### PR DESCRIPTION
**Pull request by C10udburst**
I don't think that telemetry should be mandatory.

### CODE REVIEW

**Comment by MineGame159** [comment]
There is literally nothing we can do with your UUID

**Comment by C10udburst** [comment]
Yeah, I know, but it's good practice.

**Comment by MineGame159** [comment]
It can't cause any harm, we can't use it for anything malicious, it doesn't slow the client (it has been there for a good few months), literally the only purpose of it is to make the client better.

**Comment by C10udburst** [comment]
What if Hypixel for example abuses the `http://meteorclient.com/api/online/usingMeteor` endpoint to autoban everyone using Meteor?

**Comment by MineGame159** [comment]
🤔 I didn't think of that

**Comment by MineGame159** [comment]
Ok but make it so when the setting is disable it still sends the ping but without the ?uuid parameter.

File: src/main/java/minegame159/meteorclient/gui/screens/topbar/TopBarConfig.java [hunk 1]
```diff
@@ -61,6 +61,15 @@ protected void initWidgets() {
                 .build()
         );
 
+        sgGeneral.add(new BoolSetting.Builder()
+                .name("send-data-to-api")
+                .description("If checked, your UUID will be send to Meteor's servers.")
+                .defaultValue(Config.get().sendDataToApi)
```
Comment (MineGame159) [inline src/main/java/minegame159/meteorclient/gui/screens/topbar/TopBarConfig.java:67]: Also this should be true, not the value that it is currently set to

Comment (C10udburst) [reply src/main/java/minegame159/meteorclient/gui/screens/topbar/TopBarConfig.java]: ah, ok

**Pull request merged**